Mathlib/Data/Rat/Cast/Defs.lean

```diff
@@ -47,6 +47,9 @@ theorem cast_coe_nat (n : ℕ) : ((n : ℚ) : α) = n := by
   rw [← Int.cast_ofNat, cast_coe_int, Int.cast_ofNat]
 #align rat.cast_coe_nat Rat.cast_coe_nat
 
+@[simp, norm_cast] lemma cast_ofNat (n : ℕ) [n.AtLeastTwo] :
+    ((no_index OfNat.ofNat n : ℚ) : α) = (OfNat.ofNat n : α) := by
```

Comment:
I'm slightly worried this parses as

```lean
    (((no_index OfNat.ofNat) n : ℚ) : α) = (OfNat.ofNat n : α) := by
```
And not
```suggestion
    ((no_index (OfNat.ofNat n : ℚ)) : α) = (OfNat.ofNat n : α) := by
```
which I think is what the library note recommends. I wish we had a better understanding of this